cpputil/unstructured/include/gridtools/next/mesh.hpp

```diff
@@ -24,19 +44,26 @@ namespace gridtools {
             }
 
             template <class Connectivity>
-            GT_FUNCTION auto skip_value(Connectivity const &connectivity) {
+            GT_FUNCTION auto skip_value(Connectivity const &connectivity)
```

Comment:
Not sure if `GT_FUNCTION` should be here and above. Maybe it is better always call it from host in the generated code and pass to device the result? This will simplify the modeler life: no need to do `connectivity_skip_value` cuda aware in any sense; not need even to make it fast. (it can be even not inline and belong to the separate library)

Reply:
done